specificatie/ingeschrevenpersoon.yaml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.3
+info:
+  title: IngeschrevenPersoon definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  parameters:
+    PersoonIdentificatie:
+      name: persoonIdentificatie
+      in: path
+      required: true
+      schema:
+        $ref: '#/components/schemas/PersoonIdentificatie'
+  schemas:
+    PersoonIdentificatie:
```

Comment:
moet deze als property opgenomen worden in de resource? Of is opnemen in de self-link voldoende?

Reply:
Moet niet. Consequentie is wel dat een consumer zelf de identificatie uit de url moet halen. Verder kan dan hiervoor geen uri templating worden toegepast.